task-sdk/tests/task_sdk/definitions/test_dag.py

```diff
@@ -535,7 +535,7 @@ def noop_pipeline(): ...
         assert dag.dag_id == "noop_pipeline"
 
     @pytest.mark.parametrize(
-        argnames=["dag_doc_md", "expected_doc_md"],
+        argnames=("dag_doc_md", "expected_doc_md"),
```

Comment:
Just a nitpick, for consistency's sake, can you drop the `argnames=` here so it looks like all the others, since we're going through and standardizing stuff.

```suggestion
        ("dag_doc_md", "expected_doc_md"),
```

Reply:
modify done, remove argnames= and argvalues=